src/Components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Calculator.css';
+import ClcScrn from './ClcScrn';
+import Btn from './Btn';
+/* eslint-disable react/no-array-index-key */
+class Calculator extends React.Component {
```

Comment:
- [ ] Kindly remove the `eslint-disable` statement from this file. `Eslint` helps us to write clean, readable and industry-standard codes. Using `eslint-disable` statements defeats this purpose. Kindly remove it. 👍 💻

Reply:
When I removed it, I had an error in my project.